charts/sn-platform-slim/templates/bookkeeper/bookkeeper-cluster.yaml

```diff
@@ -78,6 +78,10 @@ spec:
     {{- if .Values.bookkeeper.resources }}
     resources:
 {{ toYaml .Values.bookkeeper.resources | indent 6 }}
+    {{- end }}
+    {{- if .Values.bookkeeper.extraEnv }}
+    vars:
```

Comment:
Should we also add it to the `autoRecovery` part?

Reply:
@FushuWang PTAL the latest commit.